### DISCUSSION TITLE
Automatic Provisioning is enabled but no Development Team was selected. Please select a team or switch to Manual Provisioning from the iOS Bundle Signing page.

### DISCUSSION THREAD

**Discussion by chrfin** [dotnet/maui#3972] [Q&A] [answered]
Just installed the newest preview and connected to my Mac.
When I try to run the app on my attached iPhone I get the error from the title.
How can I configure this for .NET MAUI as the page as in Xamarin.Forms does not exist anymore/yet?

**Comment by chrfin** [comment, marked as answer]
I found a solution by using Hot Restart – that dialog adds the entries in question to the project. For Reference, the following entries are needed in the first PropertyGroup:

```
<CodesignKey>Apple Development: YOUR NAME</CodesignKey>
<CodesignProvision>VS: com.COMPANY.APPID Development</CodesignProvision>
```

You can copy these value from an old Xamarin.iOS project.

**Reply by ElliottBrand** [reply]
I'm having this same issue and adding these tags don't seem to make a difference for me. I was able to deploy from VS 2022 Preview (in Windows PC) to an iPhone connected to a Mac just fine until the past few days. I'm not sure what has suddenly caused the issue, but I did just upgrade XCode and Visual Studio 2022 Preview to the latest releases.

**Reply by chrfin** [reply]
@ElliottBrand Try also adding the following: 
`<ProvisioningType>manual</ProvisioningType>`

**Reply by ElliottBrand** [reply]
Thanks, that did bypass the error!

**Reply by Nope-Git** [reply]
Thank you, kept me from pulling all my hair out.
